core/templates/dev/head/pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html

```diff
@@ -19,6 +19,23 @@
             ng-click="reloadAllExplorations()">
       Reload All Explorations
     </button>
+    <span class="row">
+      <h3>Generate dummy explorations</h3>
+      <label for="explorations to generate">Number of explorations to generate</label>
+      <input class="form-control form-inline" style="width:50%; margin-bottom: 5px"
```

Comment:
Not done? Also, add semicolon after 5px.

(Please make absolutely sure that things are done before you say they are done.)